### PR TITLE
Some vstart fixes

### PR DESCRIPTION
**Pull request by jtlayton**
Just a couple of small cleanups and fixes for the NFS server handling in vstart to help facilitate testing. This is low priority stuff, so I didn't bother with a tracker. Tell me if it needs one.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/vstart.sh [hunk 1]
```diff
@@ -1132,28 +1138,28 @@ start_ganesha() {
...
         ganesha data = $ganesha_dir
-        pid file = $ganesha_dir/ganesha.pid
+        pid file = $ganesha_dir/ganesha-$name.pid
 EOF
 
         prun env CEPH_CONF="${conf_fn}" ganesha-rados-grace --userid $test_user -p $pool_name -n $namespace add $name
         prun env CEPH_CONF="${conf_fn}" ganesha-rados-grace --userid $test_user -p $pool_name -n $namespace
 
-        prun env CEPH_CONF="${conf_fn}" /usr/bin/ganesha.nfsd -L "$ganesha_dir/ganesha.log" -f "$ganesha_dir/ganesha.conf" -p "$ganesha_dir/ganesha.pid" -N NIV_DEBUG
+        prun env CEPH_CONF="${conf_fn}" $ganesha_path -L "$CEPH_OUT_DIR/ganesha-$name.log" -f "$ganesha_dir/ganesha-$name.conf" -p "$ganesha_dir/ganesha-$name.pid" -N NIV_DEBUG
```
Comment (varshar16) [inline src/vstart.sh:1154]: Maybe also move ganesha pid file to out dir.
```suggestion
        prun env CEPH_CONF="${conf_fn}" $ganesha_path -L "$CEPH_OUT_DIR/ganesha-$name.log" -f "$ganesha_dir/ganesha-$name.conf" -p "$CEPH_OUT_DIR/ganesha-$name.pid" -N NIV_DEBUG
```

**Comment by varshar16** [comment]
@jtlayton Yes, please create a tracker ticket for it.

File: src/vstart.sh (hunk 1 above)
Comment (jtlayton) [reply src/vstart.sh]: Good idea! Fixed.

**Comment by jtlayton** [comment]
> @jtlayton Yes, please create a tracker ticket for it.

https://tracker.ceph.com/issues/46079

**Pull request merged**